src/Ghcid.hs

```diff
@@ -135,7 +141,9 @@ autoOptions o@Options{..}
         stack <- firstJustM findStack [".",".."] -- stack file might be parent, see #62
 
         let cabal = map (curdir </>) $ filter ((==) ".cabal" . takeExtension) files
-        let opts = ["-fno-code" | null test && null run] ++ ghciFlagsRequired ++ ghciFlagsUseful
+        let opts = ["-fno-code" | null test && null run]
+                ++ ["-freverse-errors" | reversed ]
```

Comment:
Is this a good idea? We reverse the errors later on, so if we get GHC to also reverse them aren't we reversing and unreversing the errors - thus putting them back approximately as they were originally?

Reply:
As I mentioned we are not reversing them later, just the module order. But maybe I could reverse them later and not use GHC's `-freverse-error` at all. That would probably be a nicer user experience for the `--reversed` with the `-c` flag case (since we wouldn't have to specify `--ghc-options=-freverse-errors` anymore).

My reasoning was that GHC would do a better job at reversing the error messages. But if `ghcid` already has logic to split messages then it might be simpler to roll out our own reversing logic.